### DISCUSSION TITLE
Does paymentSheet only support card payments?

### DISCUSSION THREAD

**Discussion by NeatFastro** [flutter-stripe/flutter_stripe#429] [General]
I have created a payment intent with
**"payment_method_types": ["card", "ideal"]**
But the sheet only allows adding credit/debit card info, is this something that's not supported yet or I am doing something wrong?

**Comment by NeatFastro** [comment]
<img width="1440" alt="Screenshot 2021-11-08 at 1 14 39 PM" src="https://user-images.githubusercontent.com/50900820/140707722-dc5b144d-9570-4935-9db8-c588f1e7d5d5.png">

**Reply by remonh87** [reply]
did you check on your stripe dashboard : settings -> payment methods that ideal is activated? I read the docs as well and it should be there.

**Reply by remonh87** [reply]
mmm good catch I will create an issue out of it and report it to stripe

**Reply by NeatFastro** [reply]
Yeah iDEAL payment method is not only enabled but I have tested it with both live and test transactions...
That's how my current implementation looks like:
```
await Stripe.instance.confirmPayment(
        clientSecret,
        PaymentMethodParams.ideal(),
         );
```

**Comment by remonh87** [comment]
@NeatFastro  I did test it again (made mistake in payment intent) and I got the ideal option in the payment sheet. Make sure that the payment intent has ideal in it (you can check it in your stripe dashboard).

**Comment by NeatFastro** [comment]
I tried multiple time but same result, no option of iDEAL on the sheet.
This is my code:
```
const data = req.body;

    const paymentIntent = await stripe.paymentIntents.create({
      amount: data.amount,
      currency: data.currency,
      payment_method_types: ["bancontact", "card", "ideal", "sepa_debit", "sofort"],
      // payment_method_types: data.paymentMethodTypes,
      metadata: data.metaData,
      receipt_email: data.receipt_email,
      metaData: data.metaData,
    });
```
Can you please help me what am I doing wrong?
perhaps share your code?

<img width="1440" alt="Screenshot 2021-11-15 at 2 50 20 PM" src="https://user-images.githubusercontent.com/50900820/141761808-b6bb8ab8-bc74-45ff-9045-69ece972fcfc.png">

<img width="1440" alt="Screenshot 2021-11-15 at 2 50 43 PM" src="https://user-images.githubusercontent.com/50900820/141761760-b499e338-7181-4e7e-840a-38a89fbd1eb9.png">

**Reply by remonh87** [reply]
@NeatFastro when defining a payment intent like: https://github.com/flutter-stripe/flutter_stripe/blob/6789f354e199f5329a7f7c6aca4bf63a467adc46/example/lib/screens/regional_payment_methods/ideal_screen.dart#L37 .

It works for me

**Reply by NeatFastro** [reply]
ok thanks will try this.
